src/components/OTP.vue

```diff
@@ -212,8 +237,18 @@ export default {
       return this.programData.text.default.invalid.input;
     },
 
-    /** Displays message returned from OTP service */
+    /** Whether OTP message will be displayed.
+     * Will not be displayed if the timer is finished. Will be replaced by the resend OTP button
+     */
```

Comment:
```suggestion
    /** 
     * Whether to display the OTP response message.
     * It will not be displayed if the resend OTP timer is finished and will be replaced by the resend OTP button
     */
```